lib/gui/models/settings.js

```diff
@@ -34,12 +36,25 @@ const Store = require('./store');
  * settings.set('unmountOnSuccess', true);
  */
 exports.set = (key, value) => {
```

Comment:
Should this module grow a function to set multiple settings at once? Or is that something for a later PR?